assets/src/edit-story/theme.js

```diff
@@ -74,6 +75,13 @@ export const GlobalStyle = createGlobalStyle`
     border: 2px solid transparent;
     background-clip: content-box;
   }
+
+  ${({ theme }) =>
+    focusableOutlineCSS(theme.colors.accent.primary, theme.colors.bg.panel)};
+
+  :focus {
+    outline: none;
+  }
```

Comment:
This now applies to all focusable elements as a base state. It can be overridden (and has been by some form elements).

Reply:
This is an interesting idea here. Super clever. It adds the new focus style to the existing editor. My gut tells me we should not do this just yet since the design team wants the designs in the editor to be updated all at once. The exception you may have noticed is that we're using the new designs in the FTUE but that's because it can be easily wrapped and separated.